src/ado/reprun.ado

```diff
@@ -146,6 +149,24 @@ qui {
       rm_output_dir , folder("`dirout'/run2/")
     }
   }
+  
+  //display timer 
+  
+    timer off 		1
+    qui timer list 	1
+	
+    if `r(t1)' > 60 {
+      local duration :  di %9.3f `r(t1)'/60
+      noi di as res ""
+      noi di as res `"{phang}Total run time: `duration' minutes {p_end}"'
+    }
+	
+    else {
+      local duration : di %9.3f `r(t1)'
+      noi di as res ""
+      noi di as res `"{phang}Total run time: `duration' seconds {p_end}"'
+    }
+
```

Comment:
`%9.2f` is probably sufficient, but you may instead consider reporting MM:SS when it runs over 60 seconds; and HH:MM:SS when it runs over 60 minutes, as this is more standard notation and easier for the user to interpret. Reporting days is probably unnecessary.

Reply:
Updated it so now it is displayed as:

1. Seconds:
![Screenshot 2024-07-17 at 7 46 53 PM](https://github.com/user-attachments/assets/6e673adc-8c16-4bb2-8906-138267b441f2)

2. Minutes:
![Screenshot 2024-07-17 at 7 49 02 PM](https://github.com/user-attachments/assets/1d0f45f4-e761-4a70-a2a5-34399f3d5a85)

3. Hours:
![Screenshot 2024-07-17 at 7 46 21 PM](https://github.com/user-attachments/assets/bbec0eb6-2ba3-48b4-b947-105e15b6074c)

Reply:
Nice!